docs/spec_draft.md

```diff
@@ -1443,6 +1444,138 @@ specification. Numeric precision is implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.select_and_scatter
+
+### Semantics
+
+Runs a windowed selection `select` function over `operand` with shape
+`window_dimensions` and stride `window_strides`. This will produce an amount of
+selected locations whose shape matches `source`. These are then scattered to the
+output which is initialized with `init_value`.  Multiple scattered elements
+which land in the same output location are combined using the `scatter`
+function.
+
+The following diagram shows how elements in `results` are computed from
+`operand` and `source` using a concrete example.
+
+<p align="center">
+  <img src="spec_draft/select_and_scatter.png" />
```

Comment:
Maximum of blue window is 6, and maximum of pink window is 8, so shouldn't the output result be (i.e. no summing of 6+8):
[5, 6]
[7, 8]

Reply:
Thanks! There was a bug in the image where the overlapping blue and purple windows  should have a common max location. The chosen values does not represent that.

Here is what is intended in the image:
|Window | selection location in source or result for max | source value to be scatted to that location |
|--|--|--|
|green | 2,0 | source[0,0] = 5|
|gold | 3,0 | source[1,0] = 7|
|blue | 2,1 | source[0,1] = 6|
|purple | 2,1 | source[1,1] = 8|